webhook_server/libs/handlers/pull_request_handler.py

```
@@ -1677,7 +1763,9 @@ def _check_labels_for_can_be_merged(self, labels: list[str]) -> str:
         return failure_output
 
     async def skip_if_pull_request_already_merged(self, pull_request: PullRequest) -> bool:
-        if pull_request and await asyncio.to_thread(lambda: pull_request.is_merged()):
+        if pull_request and await github_api_call(
+            lambda: pull_request.is_merged(), logger=self.logger, log_prefix=self.log_prefix
```

Ruff (0.15.9)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)